server/tests/AccessTypesSpec.js

```diff
@@ -0,0 +1,212 @@
+    .then(() => {
+      db.Departments.bulkCreate([
+        testData.department1,
+        testData.department2,
+        testData.department3
+      ])
+      .then(() => {
+        db.Users.bulkCreate([
+          superAdmin,
+          departmentAdmin
+        ], { individualHooks: true })
+        .then(() => {
+          request.post('/users/login')
+          .send(superAdmin)
+          .end((err, res) => {
+            superAdminToken = res.body.token;
+            done();
+          });
+        });
+      });
+    });
+  });
+  before((done) => {
+    request.post('/users/login')
+    .send(departmentAdmin)
+    .end((err, res) => {
+      departmentAdminToken = res.body.token;
+      done();
+    });
+  });
+
+  describe('Test Default route', () => {
+    it('should check that a welcome message is returned', (done) => {
+      request.get('/')
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        expect(res.body.message)
+        .to.equal('Document Management API, Welcome');
+        done();
+      });
+    });
+  });
+  describe('Create Access Type', () => {
+    it('should check Access Type is successfully created', (done) => {
+      request.post('/accesstypes')
+      .set({ 'x-access-token': superAdminToken })
+      .send(testData.publicAccessType)
+      .end((err, res) => {
+        expect(res.status).to.equal(201);
+        expect(res.body.message)
+        .to.equal('AccessType public successfully created');
+        done();
+      });
+    });
+    it('should check Access Type already exists', (done) => {
+      request.post('/accesstypes')
+      .set({ 'x-access-token': superAdminToken })
+      .send(testData.publicAccessType)
+      .end((err, res) => {
+        expect(res.status).to.equal(409);
+        expect(res.body.message)
+        .to.equal("'public' already exists");
+        done();
+      });
+    });
+    it('should check if user is not Super Admin', (done) => {
+      request.post('/accesstypes')
+      .set({ 'x-access-token': departmentAdminToken })
+      .send(testData.publicAccessType)
+      .end((err, res) => {
+        expect(res.status).to.equal(401);
+        expect(res.body.message)
+        .to.equal('You are unauthorized to access this route');
+        done();
+      });
+    });
+    it('should check if errors are handled', (done) => {
+      request.post('/accesstypes')
+      .set({ 'x-access-token': superAdminToken })
+      .send('testData.publicAccessType')
+      .end((err, res) => {
+        expect(res.status).to.equal(400);
+        done();
+      });
+    });
+  });
+  describe('Fetch Access Type', () => {
+    it('should check Access Type is successfully fetched', (done) => {
+      request.get('/accesstypes')
+      .set({ 'x-access-token': superAdminToken })
+      .end((err, res) => {
+        expect(res.status).to.equal(200);
+        done();
+      });
+    });
+    it('should check if user is not Super Admin', (done) => {
```

Comment:
This `it block` could be more descriptive, e.g. *Should ensure only a super admin can access `/accesstypes` GET endpoint.*